the-incredible-bulk/src/main/java/gov/va/api/health/bulkfhir/service/controller/publication/NonDistributedFileWorker.java

```diff
@@ -24,20 +33,21 @@
 @Builder
 @AllArgsConstructor(onConstructor = @__({@Autowired}))
 public class NonDistributedFileWorker implements FileBuildWorker {
+
   private final DataQueryBatchClient dataQuery;
+
   private final FileClaimant claimant;
 
-  private void anonymizePatients() {
-    /* Configure the patient anonymizer and execute it here. */
-  }
+  private final BulkFileWriter fileWriter;
+
+  private final ObjectMapper jacksonMapper;
 
   @Override
   @Async(PUBLICATION_BUILD_EXECUTOR)
   public CompletableFuture<FileBuildResponse> buildFile(FileClaim claim) {
     try {
-      fetchPatients(claim);
-      anonymizePatients();
-      writePatients();
+      List<Patient> patients = fetchPatients(claim);
+      writePatients(claim, patients.stream().map(patientAnonymizer()).map(jsonStringConverter()));
```

Comment:
Should we put a `filter(Objects:nonNull)` here just to make sure that if the JSON conversion failed for some unknown reason we don't emit the string _null_.

Reply:
Yea, good call. Added that.